decentralization.adoc

```diff
@@ -2,51 +2,51 @@
 
 image::decentralization-banner.jpg[]
 
-This chapter will discuss what decentralization is and why it’s
-essential for Bitcoin to function. We’ll distinguish between the
-decentralization of miners and that of full nodes and discuss what
-they bring to the table for Bitcoin’s censorship resistance, one of
-the most central properties of Bitcoin.  The discussion then shifts
-towards understanding neutrality, or permissionlessness towards users,
-miners, and developers, which is a necessary property of a
-decentralized system. Lastly, we’ll touch upon how hard it can be to
-grasp a decentralized system like Bitcoin and present some mental
+This chapter analyzes what decentralization is and why it's
+essential for Bitcoin to function. We distinguish between the
+decentralization of miners and that of full nodes, and discuss what
+they bring to the table for censorship resistance, one of Bitcoin's
+most central properties. The discussion then shifts
+to understanding neutrality - or permissionlessness towards users,
+miners, and developers - which is a necessary property of any
+decentralized system. Lastly, we touch upon how hard it can be to
+grasp a decentralized system like Bitcoin, and present some mental
 models that might help you grok it.
 
-A system without a central point of control is referred to as being
+A system without any central point of control is referred to as being
 _decentralized_. Bitcoin is designed to avoid having a central point
-of control, or more specifically, a _central point of censorship_.
+of control, or more precisely a _central point of censorship_.
 Decentralization is a means to achieve _censorship resistance_.
 
-There are two major aspects of decentralization in Bitcoin: Miner
+There are two major aspects of decentralization in Bitcoin: miner
 decentralization and full node decentralization. Miner
-decentralization means that transaction processing isn't performed or
-coordinated by a central entity. Full node decentralization means that
-verification of the blocks, the data that miners output, is done at
-the edges of the network, ultimately by its users, and not by a few
+decentralization refers to the fact that transaction processing is not performed nor
+coordinated by any central entity. Full node decentralization refers to the fact that
+validation of the blocks, i.e. the data that miners output, gets done at
```

Comment:
I understand your view and doubts about this.

I have always heard people speaking about "block validation" so that's why I went back to that definition. However, if you think the distincition betwee "validation" and "verification" is appropriate we shall keep it in the text. You tell me!

Reply:
I think I might have used both terms throughout the book, but I've tried to use "verification". Let's stick to "verification", as long as it makes logic sense. However, the quotes might have a bias towards "validation". But I think the reader gets it.

Let's go with verification!

Reply:
If anyone else has opinions on this, now is the time to chime in. @adamjonas @jarolrod @linnearosenbaum @willcl-ark

Reply:
I usually think of verification as "checking that someone else checked it properly" (or pehaps "checking its documentation") and validation as "_doing the actual checking_ yourself".

* So I would "verify" a signature (but not generate a new one myself).
* And I would "validate" a block (i.e. perform actual checks and tests on it).

I would therefore lean towards "block validation", personally.

Reply:
Agree on block validation

Reply:
It seems everyone but me wants 'validate'. It's really hard for me to say this, but I fold. Let's use validate.